sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 7.0.0b4 (Unreleased)
+## 7.0.0b4 (2020-07-05)
```

Comment:
haha, you're not going to do the release on Sunday

Reply:
Unironically did this by adding days to today, I think I may have forgotten that the holiday weekend made today Friday.  Good call.
```suggestion
## 7.0.0b4 (2020-07-06)
```